packages/console/src/pages/Organizations/CreatePermissionModal/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import ReactModal from 'react-modal';
+
+import Button from '@/ds-components/Button';
+import FormField from '@/ds-components/FormField';
+import ModalLayout from '@/ds-components/ModalLayout';
+import TextInput from '@/ds-components/TextInput';
+import useApi from '@/hooks/use-api';
+import * as modalStyles from '@/scss/modal.module.scss';
+
+type Props = {
+  isOpen: boolean;
+  onFinish: () => void;
+};
+
+function CreatePermissionModal({ isOpen, onFinish }: Props) {
+  const api = useApi();
+  const [isLoading, setIsLoading] = useState(false);
+  const {
+    reset,
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<{ name: string; description?: string }>({ defaultValues: { name: '' } });
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  const addPermission = handleSubmit(async (json) => {
+    setIsLoading(true);
+    try {
+      await api.post('api/organization-scopes', {
+        json,
+      });
+      onFinish();
```

Comment:
nit: consider returning the response json to `onFinish()` callback, in order to reduce a `GET` request on `mutate`

Reply:
we don't have the convention to save the `GET` request currently. shall we proceed?

Reply:
OK for me